.github/ISSUE_TEMPLATE/installation_issue.md

```diff
@@ -0,0 +1,27 @@
+---
+
+name: Installation Issue 
+about: Ask about installing geopandas 
+title: ""
+labels: "installation"
+
+---
+
+- [ ] I have read the [documentation on installation](https://geopandas.org/install.html) and followed the instructions provided.
+
+---
+
+#### System information
+
+[what operating system do you have and what package management system are you
+using]
+
+#### Environment details
+
+<details>
+
+[if using conda, paste the output of `conda info` and `conda list`; if using
+pip, `pip freeze`] 
```

Comment:
In the case of pip, I'm not actually sure how likely `pip freeze` is to be useful, I just included it as the closest analog for `conda list`.